packages/files-ui/src/Components/Modules/FileBrowsers/FileInfoModal.tsx

```diff
@@ -361,6 +363,32 @@ const FileInfoModal = ({ filePath, close }: IFileInfoModuleProps) => {
                     {fullFileInfo.content?.cid}
                   </Typography>
                 </div>
+                <div className={classes.subInfoBox}>
+                  <Grid
+                    item
+                    flexDirection="row"
+                  >
+                    <Typography
+                      variant="body1"
+                      component="p"
+                    >
+                      <Trans>Decryption key:</Trans>
```

Comment:
nit (none of the other subtitles have a colon on the modal)

```suggestion
                      <Trans>Decryption key</Trans>
```